File: Backend/Backend_ProyectoFinal/Backend_ProyectoFinal/Models/TipoDeActividad.cs
namespace Backend_ProyectoFinal.Models
{
    public class TipoDeActividad
    {
        private int idTipo;
        private string nombre;
        private string descripcion;

        public TipoDeActividad(int id, string nombre, string descripcion)
        {
            this.idTipo = id;
            this.nombre = nombre;
            this.descripcion = descripcion;
        }

        public TipoDeActividad()
        {
            this.idTipo = 0;
            this.nombre = "";
            this.descripcion = "";
        }

        public int IdTipo { get => idTipo; set => idTipo = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Descripcion { get => descripcion; set => descripcion = value; }
    }
}

File: main/java/com/example/appmovil/model/TipoDeActividad.java
package com.example.appmovil.model;

public class TipoDeActividad {
    private int idTipo;
    private String nombre;
    private String descripcion;

    public TipoDeActividad(int id, String nombre, String descripcion)
    {
        this.idTipo = id;
        this.nombre = nombre;
        this.descripcion = descripcion;
    }
    public TipoDeActividad(){
        
    }

    public int getIdTipo() {
        return idTipo;
    }

    public void setIdTipo(int idTipo) {
        this.idTipo = idTipo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }


}
